docs/api/Resource.md

```diff
@@ -198,11 +207,21 @@ on network error. This can be useful to override to really customize your transp
 
 Used in `fetch()`. Resolves the HTTP [Response](https://developer.mozilla.org/en-US/docs/Web/API/Response).
 
-### static getEntitySchema() => [schema.Entity](https://github.com/ntucker/normalizr/blob/master/docs/api.md#entitykey-definition---options--)
+### static asSchema() => [Entity](./Entity)
 
-Returns the [shape of the data](https://github.com/ntucker/normalizr/blob/master/docs/api.md#schema)
-when requesting one resource at a time. Defaults to a plain object
-containing the keys. This can be useful to override if your response is in a different form.
+Returns this Resource as an [Entity](./Entity) with the TypeScript type set properly. Using
+this method instead of `this` directly key to getting correct typing from the hooks.
```

Comment:
Might want to make more like `asSchema` description. Maybe something like below

```
Returns this Resource as an `Entity` with the TypeScript type set properly. Using `asSchema()` instead of
`this` directly is key to getting correct typing from the hooks.
```